src/ontology/edits/oeo-physical.omn

```diff
@@ -14502,6 +14502,36 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1744",
         OEO_00030024
     
     
+Class: OEO_00360040
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A critical infrastructure role is a role of artificial objects, such as plants and systems, that are assessed to be crucial for the preservation of societal functions, health, safety and economical or social welfare."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000119> "The definition is based on information obtained from https://eur-lex.europa.eu/legal-content/DE/TXT/?uri=CELEX:32008L0114 and https://de.wikipedia.org/wiki/Kritische_Infrastrukturen in January 2024."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1792
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1809",
+        rdfs:label "critical infrastructure role"@en
+    
+    SubClassOf: 
+        <http://purl.obolibrary.org/obo/BFO_0000023>
+    
+    
+Class: OEO_00360041
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A critical infrastructure is an object aggregate of artificial objects and systems which have the critical infrastructure role."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000119> "The definition is based on information obtained from https://eur-lex.europa.eu/legal-content/DE/TXT/?uri=CELEX:32008L0114 and https://de.wikipedia.org/wiki/Kritische_Infrastrukturen in January 2024."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1792
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1809",
+        rdfs:label "critical infrastructure"@en
+    
+    EquivalentTo: 
+        <http://purl.obolibrary.org/obo/BFO_0000027>
+         and (<http://purl.obolibrary.org/obo/RO_0000087> some OEO_00360040)
```

Comment:
The axiom only axiomatises _A critical infrastructure is an object aggregate of artificial objects which has the critical infrastructure role_ but the definition is: _A critical infrastructure is an object aggregate **of artificial objects and systems** which have the critical infrastructure role._ I think the following axiom resembles much better the definition:
```
'critical infrastructure' EquivalentTo:
    'object aggregate'
    and ('has part' some ('artificial object' and 'has role' some 'critical infrastructure role'))
    and ('has part' some (system and 'has role' some 'critical infrastructure role'))
```